db/models/User.js

```diff
@@ -1,22 +1,20 @@
+const mongoose = require("../connection")
 const validator = require("validator")
 
-module.exports = function (mongoose) {
+//password will be hashed with bcrypt
+//so there is no need for password validation here
 
-    //password will be hashed with bcrypt
-    //so there is no need for password validation here
-    
-    const UserSchema = new mongoose.Schema({
-        email: {
-            type: String,
-            required: true,
-            unique: true,
-            validate: [validator.isEmail, "Email in invalid format"]
-        },
-        password: {
-            type: String,
-            required: true
-        }
-    });
+const UserSchema = new mongoose.Schema({
+    email: {
+        type: String,
+        required: true,
+        unique: true,
+        validate: [validator.isEmail, "Email in invalid format"]
```

Comment:
You can reuse your new error messages in error file. Otherwise add new message : D

Reply:
yikes, forgot that one 😄